Java/other/max_min_sum.java

```diff
@@ -0,0 +1,49 @@
+/*Given five positive integers, find the minimum and maximum values 
+that can be calculated by summing exactly four of the five integers. 
+Then print the respective minimum and maximum values as a single line of two space-separated long integers.*/
+import java.io.*;
+
+import java.util.*;//impoting java packages
+
+public class sol{
+  
+    public static void main(String[] args){
+      
+        Scanner in = new Scanner(System.in);
+      
+        long arr[] = new long[5];
```

Comment:
You have mentioned integers in the description but have used long here.

Reply:
Okay, will change the description then, long is used to make the code more acceptable to variety of cases.

Reply:
Hey, I have done the changes, do view it and let me know for further improvement.

Reply:
Okay.